src/storybook/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+   "presets": ["react"]
```

Comment:
Could we use the same as in the saleor:
`["@babel/preset-env", "@babel/preset-react"]`?

Reply:
we can use `["babel-preset-env", "babel-preset-react"]` cause saleor storefront using these packages.

Reply:
@piotrgrundas can we use? ["babel-preset-env", "babel-preset-react"]

Reply:
Yes, that was my suggestion.

Reply:
@piotrgrundas  Done